test/e2e/instrumentation/monitoring/stackdriver_metadata_agent.go

```diff
@@ -135,7 +136,7 @@ func verifyPodExists(response []byte, containerName string) (bool, error) {
 }
 
 func parseResource(resource interface{}) (*Resource, error) {
-	var labels map[string]string = map[string]string{}
+	var labels = map[string]string{}
```

Comment:
`labels := map[string]string{}` is the preferred pattern

Reply:
@fejta thanks. Fixed and rebased. PTAL